### PR TITLE
kevin/ENG-986

### PR DESCRIPTION
**Pull request by kcvan**
## Description
Why did you write this code?
feat(react-strap): add reactstrap, css/scss loaders to webpack, rebrand index, and test page
## How to Test
- [ ] Visit /rebrand
- [ ] Verify test page shows

Which devices did you test on?
- [x] Chrome on Mac
- [ ] Chrome on PC
- [ ] Firefox on Mac
- [ ] Firefox on PC
- [ ] Safari iPhone
- [ ] Chrome Android

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -69,7 +69,7 @@ jobs:
           name: Invalidate Cloudfront
           command: |
             if [ "${CIRCLE_BRANCH}" == "production" ]; then
-              aws cloudfront create-invalidation --distribution-id ${PRODUCTION_CLOUDFRONT_ID} --paths /index.html /admin.html
```
Comment (tc) [inline .circleci/config.yml]: can you use `work` instead of `rebrand` ?

File: config/webpack.production.js [hunk 2]
```diff
@@ -49,13 +49,19 @@ module.exports = merge(common, {
...
     }),
     new HtmlWebpackPlugin({
       filename: 'admin.html',
       template: `${root}/templates/admin.html`,
-      excludeChunks: ['index'],
+      excludeChunks: ['index', 'rebrand'],
+      ...commonHtmlWebpackOpts,
+    }),
+    new HtmlWebpackPlugin({
+      filename: 'admin.html',
```
Comment (tc) [inline config/webpack.production.js]: is this mismatched? admin and rebrand?

Comment (kcvan) [reply config/webpack.production.js]: fixed

File: .circleci/config.yml (hunk 1 above)
Comment (kcvan) [reply .circleci/config.yml]: 👍

File: src/server.js [hunk 3]
```diff
@@ -36,10 +36,12 @@ if (isDevelopmentNodeEnv) {
   );
   opn(`http://localhost:${PORT}/`);
   app.use('/admin', (req, res) => res.sendFile('templates/admin.development.html', { root: __dirname }));
+  app.use('/work', (req, res) => res.sendFile('templates/work.development.html', { root: __dirname }));
   app.use((req, res) => res.sendFile('templates/index.development.html', { root: __dirname }));
 } else {
   app.use('/js', serveStatic(path.join(__dirname, '..', 'dist', 'js')));
   app.use('/admin', serveStatic(path.join(__dirname, '..', 'dist', 'admin.html')));
+  app.use('/work', serveStatic(path.join(__dirname, '..', 'dist', 'work.html')));
   app.use((req, res) => res.sendFile(path.resolve(__dirname, '..', 'dist', 'index-v2.html')));
```
Comment (tc) [inline src/server.js]: can you rename index-v2 to index.html

Comment (kcvan) [reply src/server.js]: 👍

**Pull request merged**